ReactWindows/ReactNative.Shared.Tests/Bridge/NativeModuleWrapperBaseTests.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using ReactNative.Bridge;
+using System;
+using System.Collections.Generic;
+
+namespace ReactNative.Tests.Bridge
+{
+    [TestFixture]
+    public class NativeModuleWrapperBaseTests
+    {
+        [Test]
+        public void NativeModuleWrapperBase_ArgumentChecks()
+        {
+            AssertEx.Throws<ArgumentNullException>(() => new TestWrapper(null), ex => Assert.AreEqual("module", ex.ParamName));
```

Comment:
the war continues... ;)

please use NUnit here like we did for all other new tests. all issues with running mixed tests were fixed in VS2017 15.6.4 (but always worked with the superior ReSharper test runner anyway).

we agreed last year this formatting of expected exceptions is easier to ready (from ReactBridgeTests):
https://github.com/Microsoft/react-native-windows/blob/master/ReactWindows/ReactNative.Shared.Tests/Bridge/ReactBridgeTests.cs#L29